index.html

```diff
@@ -958,14 +1056,200 @@ <h4 id="signed-raw">Signed Raw Transaction</h4>
+
+      // Add outputs to the Tx
+      for (const output of outputs) {
+        cTx.addoutput(output[0], output[1]);
+      }
+
+      // Sign and broadcast!
+      if (!masterKey.isHardwareWallet) {
+        const sign = await cTx.sign(masterKey, 1);
+        sendTransaction(sign);
+      } else {
+        // Format the inputs how the Ledger SDK prefers
+        const arrInputs = [];
+        const arrAssociatedKeysets = [];
+        for (const cInput of cTx.inputs) {
+          const cInputFull = await getTxInfo(cInput.outpoint.hash);
+          arrInputs.push([await cHardwareWallet.splitTransaction(cInputFull.hex), cInput.outpoint.index]);
+          arrAssociatedKeysets.push(cInput.path);
+        }
+        const cLedgerTx = await cHardwareWallet.splitTransaction(cTx.serialize());
+              const strOutputScriptHex = await cHardwareWallet
+          .serializeTransactionOutputs(cLedgerTx)
+          .toString("hex");
+
+        // Sign the transaction via Ledger
+        const strSerialisedTx = await confirmPopup(
+          {
+            title: "Confirm your transaction",
+            html: createTxConfirmation(outputs),
+            resolvePromise: cHardwareWallet.createPaymentTransactionNew({
+              inputs: arrInputs,
+              associatedKeysets: arrAssociatedKeysets,
+              outputScriptHex: strOutputScriptHex,
+            }),
+          }
+        );
+
+        // Broadcast the Hardware (Ledger) TX
+        sendTransaction(strSerialisedTx);
+        
+      }
+      let masternode_privateKey= await generateMnPrivkey();
+       await confirmPopup({
+          title: "Your Masternode Private Key",
+          html: `${masternode_privateKey} <br> Save this private key and copy it on your VPS`, 
+        });
+        createAlert("success","Your masternode has been created! Wait 15 confirmations before accessing it") 
+    }
+    async function importMasternode(){ 
+      const mnPrivKey = domMnPrivateKey.value;
+      
+      const ip = domMnIP.value;
+      let address;
+      let collateralTxId;
+      let outidx;
+      let collateralPrivKey;
+      domMnIP.value = "";
+      domMnPrivateKey.value = "";
+
+      if (!ip.includes(":")) {
+	address = `${ip}:${cChainParams.MASTERNODE_PORT}`;
+      } else {
+	address = ip;
+      }
+
+      if (!masterKey.isHD) {
+        //the first UTXO should have 10k PIV
+        let firstUTXO=cachedUTXOs[cachedUTXOs.length-1];
+
+        collateralTxId = firstUTXO.id;
+        outidx = firstUTXO.vout;
+        collateralPrivKey = await masterKey.getPrivateKey();
+      } else {
+        let path = domMnTxId.value;
+        let collateraInSats=10000*COIN; //collateral in satoshi units
+	console.log(path);
+        const masterUtxo = cachedUTXOs.findLast(u=>u.path === path); // first UTXO for each address in HD
+	console.log(masterUtxo);
+        //sanity check:
+        if(masterUtxo.sats!==10000*COIN){
+          createAlert("warning", "The UTXO doesn't match", 10000);
```

Comment:
Could be a bit more clear, perhaps "This is not a suitable UTXO for a Masternode"